### DISCUSSION TITLE
How to use widgets?

### DISCUSSION THREAD

**Discussion by mac-joomla** [templaza/astroid-framework#529] [Q&A] [answered]
Hello, I like these new widgets, but what way they are supposed to use?

If I add a widget then it appears on all pages. Do I have to use different templates with different widgets? Or ist it possible to use widgets somehow in editing a Joomla article?

Thank you and best regards

**Comment by sonvnn** [comment, marked as answer]
At the moment, You have to use different template to display different widgets layout.

Thanks & Best Regards,
Sonny

**Comment by Chacapamac** [comment]
@sonvnn 
These widget are really cool and it make sense that they are connected to one specific layout.
I see that you have the infos on them on Astroid Blog (GREAT). 
At one point you saying “You can add these new widgets directly into the Astroid's layout, they carry the same functions as add-ons of the page builder do.” 
**What “Page Builder” are you talking about?**
_Good Day Sonny!_

**Reply by sonvnn** [reply]
I mean its feature like a Page Builder. It is unique not like any extension.

**Reply by Chacapamac** [reply]
Oh, I see @sonvnn . Can these “Widgets” be easily transform in Joomla Plugins? Just a thought?
